apps/extension/src/routes/popup/home/inspect-address/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from '@penumbra-zone/ui/components/ui/box';
+import { IconInput } from '@penumbra-zone/ui/components/ui/icon-input';
+import { useEffect, useState } from 'react';
+import { MagnifyingGlassIcon } from '@radix-ui/react-icons';
+import { Result as TResult } from './types';
+import { getResultFromBech32mAddress } from './get-result-from-bech32m-address';
+import { Result } from './result';
+
+export const InspectAddress = () => {
+  const [address, setAddress] = useState('');
+  const [result, setResult] = useState<TResult | undefined>();
+
+  useEffect(() => {
+    void getResultFromBech32mAddress(address).then(setResult);
+  }, [address]);
+
+  return (
+    <Box label='Inspect address'>
```

Comment:
Can we have a border color that changes that would indicate an input is not valid? It would be nice to get some type of response from the UI if I put an invalid address.

Reply:
I like it! Here's what I came up with:

![screencap](https://github.com/penumbra-zone/web/assets/1121544/5baea0e6-13bf-46d7-9476-34d85d942ff6)